### PR TITLE
Refactor locale util

### PR DESCRIPTION
**Pull request by konifar**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -78,6 +78,7 @@ android {
                     mapsApiKey: System.getenv('ANDROID_MAPS_API_KEY') != null ? System.getenv('ANDROID_MAPS_API_KEY') : android_maps_api_key
             ]
             buildConfigField("String", "GA_TRACKING_ID", "\"${ga_tracking_id_debug}\"")
+            buildConfigField "String", "CONFERENCE_TIMEZONE", "\"Asia/Tokyo\""
```
Comment (konifar) [inline app/build.gradle]: it is preparation of the next function that user can show session time as local time.

File: app/src/androidTest/java/io/github/droidkaigi/confsched/util/AppUtilsTest.java [hunk 2]
```diff
@@ -25,24 +20,4 @@ public void testGitHubUrl() throws Exception {
         assertThat(AppUtil.getGitHubUrl("konifar")).isEqualTo("https://github.com/konifar");
     }
 
-    @Test
-    public void testCurrentLanguageId() throws Exception {
```
Comment (konifar) [inline app/src/androidTest/java/io/github/droidkaigi/confsched/util/AppUtilsTest.java]: Remove to LocaleUtilsTest

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -8,6 +8,7 @@
     <application
         android:name=".MainApplication"
         android:allowBackup="true"
+        android:configChanges="locale"
```
Comment (konifar) [inline app/src/main/AndroidManifest.xml]: It is necessary to change `Locale.getDefault()`. I forgot this setting.

File: app/src/main/java/io/github/droidkaigi/confsched/util/AppUtil.java [hunk 4]
```diff
@@ -50,53 +42,6 @@ public static String getFacebookUrl(@NonNull String name) {
         return FACEBOOK_URL + name;
     }
 
-    public static void initLocale(Context context) {
-        setLocale(context, getCurrentLanguageId(context));
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/util/AppUtil.java]: These utility methods were moved to `LocaleUtill`.

**Comment by KeithYokoma** [comment]
[LGTM](http://lgtm.in/i/kagNDcniq)

[:+1:](http://lgtm.in/u/kagNDcniq)

File: app/src/main/java/io/github/droidkaigi/confsched/util/DateUtil.java [hunk 5]
```diff
@@ -47,13 +40,15 @@ public static String getHourMinute(Date date, Context context) {
...
-        Locale locale = AppUtil.getCurrentLocale(context);
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
-            String pattern = DateFormat.getBestDateTimePattern(locale, FORMAT_YYYYMMDDKKMM);
-            return new SimpleDateFormat(pattern, locale).format(date);
+            String pattern = DateFormat.getBestDateTimePattern(Locale.getDefault(), FORMAT_YYYYMMDDKKMM);
+            return new SimpleDateFormat(pattern).format(date);
         } else {
-            SimpleDateFormat sdf = (SimpleDateFormat) java.text.DateFormat.getDateInstance(java.text.DateFormat.LONG, locale);
-            return sdf.format(date) + getHourMinute(date, context);
+            java.text.DateFormat dayOfWeekFormat = SimpleDateFormat.getDateInstance(SimpleDateFormat.LONG);
```
Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/util/DateUtil.java]: Why do you use SimpleDateFormat? getDateInstance is taken over from DateFormat right? :laughing:

**Comment by hotchemi** [comment]
LGTM!

File: app/src/main/java/io/github/droidkaigi/confsched/util/DateUtil.java (hunk 5 above)
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/util/DateUtil.java]: You are right. I'll fix it! Thanks!

File: app/build.gradle (hunk 1 above)
Comment (ytRino) [inline app/build.gradle]: Dont' you declare at `defaultConfig` block?

Comment (konifar) [inline app/build.gradle]: Oh, I don't know that. Could you tell me detail? :bow:

Comment (ytRino) [inline app/build.gradle]: Just move it to 
```
defaultConfig{
  ...
  buildConfigField "String", "CONFERENCE_TIMEZONE", "\"Asia/Tokyo\""
}
```
for defaults.

Comment (konifar) [inline app/build.gradle]: Good! I'll fix as you said

**Pull request merged**